src/Subscribers.php

```diff
@@ -291,6 +292,30 @@ public function getSubscribers(EntityInterface $entity, MessageInterface $messag
   }
 
   /**
+   * Helper method to determine if the given entity belongs to the given user.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity to check ownership of.
+   * @param int $uid
+   *   The user ID to check for ownership.
+   *
+   * @return bool
+   *   Returns TRUE if the entity is owned by the given user ID.
+   */
+  protected function isEntityOwner(EntityInterface $entity, $uid) {
```

Comment:
Potentially. It might be helpful in the [workflow initiative](https://www.drupal.org/node/2721129)...